pkgs/development/python-modules/pex/default.nix

```diff
@@ -0,0 +1,28 @@
+{lib, buildPythonPackage, fetchPypi, setuptools, wheel, requests}:
+
+buildPythonPackage rec {
+  name = "pex-${version}";
+  version = "1.4.5";
+
+  src = fetchPypi {
+    pname  = "pex";
+    sha256 = "04s9qvx87ngfs3m91qsrmk0ll16s0ldrvlxjbg4y1ic4bgsjq3hj";
+    inherit version;
+  };
+
+  propagatedBuildInputs = [ setuptools wheel requests ];
+
```

Comment:
Blind guess:

```nix
checkInputs = [ pytest ];
```

to fix the build issues:

Reply:
Thanks, I'll give it a try..